server/src/http/controllers/application.controller.ts

```diff
@@ -15,6 +16,56 @@ const rateLimitConfig = {
 } as const
 
 export default function (server: Server) {
+  server.post(
+    "/v1/application",
+    {
+      schema: zRoutes.post["/v1/application"],
+      config: {
+        rateLimit: {
+          max: 5,
+          timeWindow: "5s",
+        },
+      },
+      bodyLimit: 5 * 1024 ** 2, // 5MB
+    },
+    async (req, res) => {
+      const { company_type } = req.body
+
+      const convertedCompanyType = () => {
```

Comment:
```suggestion
      const convertedCompanyType = (company_type) => {
```
pour la rendre pure pour pouvoir la sortir du controller